jubilant/_juju.py

```diff
@@ -1277,6 +1392,24 @@ def _temp_dir(self) -> str:
             return tempfile.gettempdir()
 
 
+def _base_to_series(base: str) -> str:
+    """Convert a base to a series name."""
+    name, cycle = base.split('@', 1)
+    if name != 'ubuntu':
+        raise ValueError(f'base must be an Ubuntu base, not {name!r}')
+    return {
+        '14.04': 'trusty',
+        '16.04': 'xenial',
+        '18.04': 'bionic',
+        '20.04': 'focal',
+        '22.04': 'jammy',
+        '24.04': 'noble',
+        '24.10': 'oracular',
+        '25.04': 'plucky',
+        '25.10': 'questing',
+    }[cycle]
```

Comment:
Style nit: I would prefer this "constant" dict was extracted to a module-lev var to avoid building it every time. `_SERIES_NAMES` or `_SERIES_BY_BASE` or something.